samples/docker/enable-multus/rke2controlplane-test.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: controlplane.cluster.x-k8s.io/v1alpha1
+kind: RKE2ControlPlane
+metadata:
+  name: rke2cp-test
+  namespace: webhook-test
+spec: 
+  replicas: 3
+  agentConfig:
+    version: v1.24.11+rke2r1
+  serverConfig:
+    cniMultusEnable: true
+      #cni: calico
```

Comment:
this should be uncommented, right?

Reply:
Well, both are fine. Maybe for our users I should uncomment, but an empty `cni` field with `cniMultusEnable` to `true` was an edge case that I tested.

Reply:
I was just making sure we either remove or uncomment it not to confuse the users

Reply:
Done. :)